e2e/pages/createCase/createCase.js

```diff
@@ -7,12 +7,23 @@ module.exports = {
     caseType: 'case-type',
     event: 'event',
   },
+  enterCaseNamePage: {
+    caseName: '#caseName',
+  },
+  testCaseName: 'Barnet Council v Smith',
+  continueButton: 'Continue',  
   startButton: 'Start',
 
+  enterCaseName() {
```

Comment:
Please introduce argument to the method that takes case name.
```suggestion
  enterCaseName(caseName = 'Barnet Council v Smith') {
```